stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -2830,6 +2830,10 @@ internal constructor(
             pluginFactories.add(pluginFactory)
         }
 
+        public fun withPlugins(vararg pluginFactoryArgs: PluginFactory): Builder = apply {
```

Comment:
Do we need the `withPlugin(pluginFactory: PluginFactory)`  method?
As far as this new one accepts a `vararg` arguments, we can provide only this one.